kivy/lang/builder.py

```diff
@@ -865,7 +865,10 @@ def dump_builder_stats():
             '</style>']
         files = set([x[1].ctx.filename for x in Builder.rules])
         for fn in files:
-            lines = open(fn).readlines()
+            try:
+                lines = open(fn).readlines()
```

Comment:
This should really be `with open(fn) as fh: fh.readlines()`. Also, rather than excepting everything, you should only except IOError.